src/blog/sparkplug.md

```diff
@@ -0,0 +1,188 @@
+---
+title: 'Sparkplug — A non-optimizing JavaScript compiler'
+author: '[Leszek Swirski](https://twitter.com/leszekswirski) — maybe not the brightest spark, but at least the fastest one'
+avatars:
+  - leszek-swirski
+date: 2021-05-27
+tags:
+  - JavaScript
+extra_links:
+  - href: https://fonts.googleapis.com/css?family=Gloria+Hallelujah&display=swap
+    rel: stylesheet
+description: 'In V8 v9.1 we''re improving V8 performance by 5-15% with Sparkplug: a new, non-optimizing JavaScript compiler.'
```

Comment:
We use correct apostrophes instead, which incidentally also helps avoid escaping.

```suggestion
description: 'In V8 v9.1 we’re improving V8 performance by 5-15% with Sparkplug: a new, non-optimizing JavaScript compiler.'
```

Reply:
(please also replace other occurences of `'` used as apostrophe in the blog post)

Reply:
Done